### ISSUE TITLE
News errors with HA 2022.4.0

### ISSUE THREAD

**Issue by Mick51** [al-one/hass-xiaomi-miot#505]
Logger: homeassistant.setup
Source: setup.py:298
First occurred: 00:11:30 (1 occurrences)
Last logged: 00:11:30

Unable to prepare setup for platform xiaomi_miot_raw.fan: Platform not found (cannot import name 'ATTR_SPEED' from 'homeassistant.components.fan' (/usr/src/homeassistant/homeassistant/components/fan/__init__.py)).

Logger: homeassistant.components.sensor
Source: helpers/entity_registry.py:383
Integration: Capteur (documentation, issues)
First occurred: 00:11:51 (1 occurrences)
Last logged: 00:11:51

Error adding entities for domain sensor with platform xiaomi_miot
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 382, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 510, in _async_add_entity
    entry = entity_registry.async_get_or_create(
  File "/usr/src/homeassistant/homeassistant/helpers/entity_registry.py", line 383, in async_get_or_create
    raise ValueError("entity_category must be a valid EntityCategory instance")
ValueError: entity_category must be a valid EntityCategory instance

Cette erreur provient d'une intégration personnalisée

Logger: custom_components.xiaomi_miot_raw.deps.xiaomi_cloud_new
Source: custom_components/xiaomi_miot_raw/deps/xiaomi_cloud_new.py:181
Integration: Xiaomi MIoT (documentation, issues)
First occurred: 00:11:31 (1 occurrences)
Last logged: 00:11:31

Timeout while loading MiCloud device list

**Comment by al-one** [comment]
#503

**Comment by al-one** [comment]
> ValueError: entity_category must be a valid EntityCategory instance

- #504
- Upgrade to master branch.

**Comment by Krzysztonek** [comment]
Hi,
Same issues here:

![image](https://user-images.githubusercontent.com/87784133/162421334-896f1e93-1348-4906-9d19-1dea67a73dbb.png)

 Logger: homeassistant.setup
Source: setup.py:298
First occurred: 11:46:58 (6 occurrences)
Last logged: 11:46:58

`    Unable to prepare setup for platform xiaomi_miot.fan: Platform not found (cannot import name 'SPEED_OFF' from 'homeassistant.components.fan' (/usr/src/homeassistant/homeassistant/components/fan/__init__.py)).
    Unable to prepare setup for platform xiaomi_miot.climate: Platform not found (cannot import name 'SPEED_OFF' from 'homeassistant.components.fan' (/usr/src/homeassistant/homeassistant/components/fan/__init__.py)).
    Unable to prepare setup for platform xiaomi_miot.cover: Platform not found (cannot import name 'SPEED_OFF' from 'homeassistant.components.fan' (/usr/src/homeassistant/homeassistant/components/fan/__init__.py)).
    Unable to prepare setup for platform xiaomi_miot.humidifier: Platform not found (cannot import name 'SPEED_OFF' from 'homeassistant.components.fan' (/usr/src/homeassistant/homeassistant/components/fan/__init__.py)).
    Unable to prepare setup for platform xiaomi_miot.device_tracker: Platform not found (cannot import name 'SPEED_OFF' from 'homeassistant.components.fan' (/usr/src/homeassistant/homeassistant/components/fan/__init__.py)).`

**Comment by ibl129** [comment]
The same over here. Integration is loaded as it shows in the integration overview nevertheless it does not load any entity it finds by configuring the integration.

**Comment by danielsitnik** [comment]
Same here.. just upgraded to the master branch and have the same errors as @Krzysztonek describe above.

I'm using Home Assistant 2022.4.1.

```
Logger: homeassistant.setup
Source: setup.py:298
First occurred: April 8, 2022, 23:17:35 (6 occurrences)
Last logged: April 8, 2022, 23:17:35

Unable to prepare setup for platform xiaomi_miot.device_tracker: Platform not found (cannot import name 'SPEED_OFF' from 'homeassistant.components.fan' (/usr/src/homeassistant/homeassistant/components/fan/__init__.py)).
Unable to prepare setup for platform xiaomi_miot.fan: Platform not found (cannot import name 'SPEED_OFF' from 'homeassistant.components.fan' (/usr/src/homeassistant/homeassistant/components/fan/__init__.py)).
Unable to prepare setup for platform xiaomi_miot.cover: Platform not found (cannot import name 'SPEED_OFF' from 'homeassistant.components.fan' (/usr/src/homeassistant/homeassistant/components/fan/__init__.py)).
Unable to prepare setup for platform xiaomi_miot.climate: Platform not found (cannot import name 'SPEED_OFF' from 'homeassistant.components.fan' (/usr/src/homeassistant/homeassistant/components/fan/__init__.py)).
Unable to prepare setup for platform xiaomi_miot.humidifier: Platform not found (cannot import name 'SPEED_OFF' from 'homeassistant.components.fan' (/usr/src/homeassistant/homeassistant/components/fan/__init__.py)).
```

**Comment by al-one** [comment]
Fixed in v0.5.6.
